test/unit/helpers/Fixture.js

```diff
@@ -23,7 +23,7 @@ export default class Fixture {
         this.minter = await this.deployAndRegister(MinterMock, "Minter")
         this.bondingManager = await this.deployAndRegister(BondingManagerMock, "BondingManager")
         this.roundsManager = await this.deployAndRegister(GenericMock, "RoundsManager")
-        this.ticketBroker = await this.deployAndRegister(GenericMock, "TicketBroker")
+        this.jobsManager = await this.deployAndRegister(GenericMock, "JobsManager")
```

Comment:
In continuation of my comment above, to avoid confusion, I would then change this to:

```
this.ticketBroker = await this.deployAndRegister(GenerickMock, "TicketBroker")
// Register TicketBroker as both TicketBroker and JobsManager
await this.controller.setContractInfo(contractId("JobsManager"), this.ticketBroker.address, web3.utils.asciiToHex("0x123"))
```

We can then also update the tests to use `fixture.ticketBroker` so that the semantics reflect the real logic.

Reply:
Fixed in c7d0ee4